runner/Cargo.toml

```diff
@@ -12,3 +12,5 @@ toml = "0.8.10"
 benchmark_analyzer = { path = "../benchmark_analyzer" }
 config_helpers = { path = "../crates/config_helpers" }
 walkdir = "2"
+env_logger = "0.11.5"
+log = "0.4"
```

Comment:
Replace with `log = { workspace = true }`, this will use the same version as all the other crates, and hopefully save a bit of compile time :)